pkg/sdk/agent.go

```diff
@@ -27,15 +27,17 @@ type SDK interface {
 	Data(ctx context.Context, dataset *os.File, filename string, privKey any) error
 	Result(ctx context.Context, privKey any, resultFile *os.File) error
 	Attestation(ctx context.Context, reportData [size64]byte, nonce [size32]byte, attType int, attestationFile *os.File) error
+	IMAMeasurements(ctx context.Context, privKey any, resultFile *os.File) ([]byte, error)
```

Comment:
we can leave this unauthenticated as we do in attestation. no need for private key